src/modules/twap/containers/TwapConfirmDetails/LimitPriceRow.tsx

```diff
@@ -0,0 +1,22 @@
+import { Currency } from '@uniswap/sdk-core'
```

Comment:
This component and `MinReceivedRow` are right now pretty much the same. The idea was to have one separate component for each row, but I haven't decided yet how to do this.